### ISSUE TITLE
What is the final model used in the paper?

### ISSUE THREAD

**Issue by codeslake** [HuiZeng/Grid-Anchor-based-Image-Cropping-Pytorch#3]
Hi, Thanks for your fabulous work!

I wonder what is the final model used in the paper, "Reliable and Efficient Image Cropping: A Grid Anchor based Approach Hui".

It seems like vgg16 is used in the paper, but, what is the exact training command?
Specifically, I want to find out options for, scale, downsample, augmentation and reduced dim.

Thanks.
